.idea/runConfigurations/android.xml

```diff
@@ -1,33 +0,0 @@
-<component name="ProjectRunConfigurationManager">
```

Comment:
I thought this was necessary for fixing the fixed listview sizing, which I needed for testing (saw this in Tom's commit (Fixed listview sizing in Main_Activity)). Tested it and it wasn't necessary, so i reverted it.